campaigns/inferno_steadwicks_fall.tex

```diff
@@ -205,11 +205,11 @@ \subsection*{\MakeUppercase{Additional Rules}}
       Gate'' location (only if you have the ``Castle Gate'' built in your Town).
 \end{itemize}
 
-\vspace*{\fill}
+% \vspace*{\fill}
 \begin{center}
-  \framedimage[0.9\linewidth]{\art/demon.jpg}
+  \framedimage[\columnwidth]{\art/demon.jpg}
```

Comment:
```suggestion
  \framedimage[0.95\columnwidth]{\art/demon.jpg}
```
Please make it slightly smaller. Also, please delete the surrounding commented out vspaes.

Reply:
At full columnwidth those vspaces made for an extra page break somehow. If they don't with 0.95 width, I'll restore them ofc.

Reply:
Please do.